frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,95 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-user-preferences"
+              And user is at Add page
+
+
+        @smoke @to-do
+        Scenario: Access user preference from the masthead: UP-01-TC01
+             When user clicks on user dropdown
+             Then user will see User Preference
```

Comment:
I think we can remove this scenario. In below scenario, 
'And user clicks User Preference option' - covers this scenario